CorsixTH/Lua/objects/bed.lua

```diff
@@ -65,10 +65,10 @@ object.usage_animations = {
 
 local anim_mgr = TheApp.animation_manager
 local kf1, kf2, kf3 = {1, -1}, {0.4, -1}, {-0.3, -1}
```

Comment:
Not you, but do you know what the `kfX` variables are meant to stand for?

Reply:
Ah, it's keyframe...

Reply:
A key-frame is a relative frame number and a marker position.
So this is actually a tile position, so names aren't that good.